test/Elastic.Apm.Tests.Utilities/XUnit/MemberData.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+using System;
+using System.Collections.Generic;
+
+namespace Elastic.Apm.Tests.Utilities.XUnit;
+
+public static class MemberData
+{
+	public static IEnumerable<object[]> TestWithDiagnosticSourceOnly()
```

Comment:
We already skip tests based on runtime: https://github.com/elastic/apm-agent-dotnet/blob/main/test/Elastic.Apm.Tests/NetCoreAndNetFact.cs (similar, but not the same).

Maybe you have seen it - just pointing to it, since it may be relevant.

Reply:
Thanks for pointing it out. Might be useful along the way.